tests/Etc/CacheManagerService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Tests\Etc;
+
+use Illuminate\Cache\CacheManager;
+use Illuminate\Cache\Repository;
+
+class CacheManagerService
+{
+    public Repository|null $cache = null;
```

Comment:
I think the `|null` can be removed here
```suggestion
    public Repository $cache = null;
```